docs/getting-started/release-notes/v2_5_0.md

```diff
@@ -42,25 +42,24 @@ Zowe Version 2.5.0 contains the enhancements that are described in the following
 
 #### Zowe CLI (Core)
 
-- Added new functions to support the changing of a job class and the hold status of a job. Can now call `zowe jobs modify job [jobid]` with options `--jobclass`, `--hold` and `--release`. [#1156](https://github.com/zowe/zowe-cli/issues/1156)
+- Added new functions to support the changing of a job class and the hold status of a job. Can now call `zowe jobs modify job [jobid]` with options `--jobclass`, `--hold`, and `--release`. ([#1156](https://github.com/zowe/zowe-cli/issues/1156))
 - Updated Imperative to incorporate new `zowe config report-env` command from version `5.7.0`.
-- Added design documentation for roadmap feature to store secure properties in memory.
-- Allow `zowe files view ds ... --range SSS-EEE | SSS,NNN`. [#1539](https://github.com/zowe/zowe-cli/issues/1539)
-- Added `ZosFilesCreateOptions.alcunit` option to PDS definition. [#1203](https://github.com/zowe/zowe-cli/issues/1203)
+- To view a subset of data set content instead of the entire file, added new command `zowe files view ds ... --range SSS-EEE | SSS,NNN`. ([#1539](https://github.com/zowe/zowe-cli/issues/1539))
+- To define the unit of space allocation when creating a data set, added `ZosFilesCreateOptions.alcunit` option to PDS definition. ([#1203](https://github.com/zowe/zowe-cli/issues/1203))
 
 <!--7.7.0, 7.8.0, 7.9.0-->
 
 #### Zowe CLI Imperative Framework
 
-- Enhancement: Exported `AppSettings` for cli and other apps to use [#840](https://github.com/zowe/imperative/issues/840)
-- Enhancement: Add `zowe config report-env` command to show a diagnostic report of the CLI's working environment.
-- Extend zowe plugins verbs to show information for a plugin's first steps [#1325](https://github.com/zowe/zowe-cli/issues/1325)
+- Exported `AppSettings` for CLI and other applications to use ([#840](https://github.com/zowe/imperative/issues/840))
+- To view a diagnostic report of the CLI working environment, added the `zowe config report-env` command.
+- To show information for a plug-in's first steps, added the `show-first-steps` command. ([#1325](https://github.com/zowe/zowe-cli/issues/1325))
```

Comment:
Minor detail:
I'm curious about the desired format for these changelog entries.
1. `Added XYZ to do ABC`
2. `To do ABC, added XYZ`

Reply:
@zFernand0 -- I'm playing around w/ formatting. We don't seem to have a standard format across all repos, and this is something that the doc squad has discussed as an issue we need to address.